apps/dashboard/src/components/primitives/editor.tsx

```diff
@@ -16,9 +18,91 @@ const editorVariants = cva('-mx-1 -mt-[2px] h-full w-full flex-1 [&_.cm-focused]
   },
 });
 
+const baseTheme = EditorView.baseTheme({
+  '&light': {
+    backgroundColor: 'transparent',
+  },
+  '.cm-tooltip-autocomplete .cm-completionIcon-variable': {
+    '&:after': {
+      content: "''",
+      height: '16px',
+      width: '16px',
+      display: 'block',
+      backgroundRepeat: 'no-repeat',
+      backgroundImage: `url('${functionIcon}')`,
+    },
+  },
+  '.cm-tooltip-autocomplete.cm-tooltip': {
+    position: 'relative',
+    overflow: 'hidden',
+    borderRadius: 'var(--radius)',
+    border: '1px solid var(--neutral-100)',
+    backgroundColor: 'hsl(var(--background))',
+    boxShadow: '0px 1px 3px 0px rgba(16, 24, 40, 0.10), 0px 1px 2px 0px rgba(16, 24, 40, 0.06)',
+    '&:before': {
+      content: "''",
+      top: '0',
+      left: '0',
+      right: '0',
+      height: '30px',
+      display: 'block',
+      backgroundRepeat: 'no-repeat',
+      backgroundImage: `url('${autocompleteHeader}')`,
+    },
+    '&:after': {
+      content: "''",
+      bottom: '30px',
+      left: '0',
+      right: '0',
+      height: '30px',
+      display: 'block',
+      backgroundRepeat: 'no-repeat',
+      backgroundImage: `url('${autocompleteFooter}')`,
+    },
```

Comment:
What should catch your eyes are two pseudo-elements and two background images :P this is the hacky way to adjust it to the design requirements. Basically, the autosuggestions header and footer are svgs :P 
I did it this way because there is no way to easily adapt the component to the requirements, and I didn't want to spend days on it.